dataframe-geo/build.gradle.kts

```diff
@@ -0,0 +1,73 @@
+import org.jetbrains.kotlin.gradle.tasks.BaseKotlinCompile
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    with(libs.plugins) {
+        alias(kotlin.jvm)
+        alias(publisher)
+        alias(jupyter.api)
+    }
+}
+
+group = "org.jetbrains.kotlinx"
+
+repositories {
+    // geo repositories should come before Maven Central
+    maven("https://maven.geotoolkit.org")
+    maven("https://repo.osgeo.org/repository/release")
+    mavenCentral()
+}
+
+// https://stackoverflow.com/questions/26993105/i-get-an-error-downloading-javax-media-jai-core1-1-3-from-maven-central
+// jai core dependency should be excluded from geotools dependencies and added separately
+fun ExternalModuleDependency.excludeJaiCore() = exclude("javax.media", "jai_core")
+
+
+dependencies {
+    api(project(":core"))
+
+    implementation(libs.geotools.main) { excludeJaiCore() }
+    implementation(libs.geotools.shapefile) { excludeJaiCore() }
+    implementation(libs.geotools.geojson) { excludeJaiCore() }
+    implementation(libs.geotools.referencing) { excludeJaiCore() }
+    implementation(libs.geotools.epsg.hsql) { excludeJaiCore() }
+
+    implementation(libs.jai.core)
+
+    implementation(libs.jts.core)
+    implementation(libs.jts.io.common)
+
+    implementation(libs.ktor.client.core)
+    implementation(libs.ktor.client.cio)
+    implementation(libs.ktor.client.content.negotiation)
+    implementation(libs.ktor.serialization.kotlinx.json)
+
+}
+
+tasks.withType<KotlinCompile>().configureEach {
+    val friendModule = project(":core")
+    val jarTask = friendModule.tasks.getByName("jar") as Jar
+    val jarPath = jarTask.archiveFile.get().asFile.absolutePath
+    (this as BaseKotlinCompile).friendPaths.from(jarPath)
+}
+
+kotlinPublications {
+    publication {
+        publicationName = "dataframeGeo"
+        artifactId = "dataframe-geo"
+        description = "GeoDataFrame API"
+        packageName = artifactId
+    }
+}
+
+tasks.processJupyterApiResources {
+    libraryProducers = listOf("org.jetbrains.kotlinx.dataframe.jupyter.IntegrationGeo")
+}
+
+
+tasks.test {
+    useJUnitPlatform()
+}
+kotlin {
+    jvmToolchain(11)
```

Comment:
We have a global problem - DataFrame uses 8 and the Jupyter API uses 11. Therefore, in the future it will be necessary to allocate separate module(s) for integrations.

Reply:
https://github.com/Kotlin/dataframe/issues/775

Reply:
but for now, for convenience, I'd keep it 11

Reply:
Actually, you can probably remove this piece, as `kotlin.jvmToolChain(11)` is already called from the main `build.gradle.kts` file; It configures all subprojects automatically to target jvm 8 in the published jars while setting the jvm toolchain (so the version gradle uses to build and test the project) to 11.

Reply:
ok